packages/extension-vscode/scripts/tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Why do we need a new `tsconfig.json` under `scripts`? Is it for `checkJS`? If so, can't we just add it to the top of the file (or maybe add that option in the root of our project?)

Reply:
Yes, this is for `checkJS`. We've used this across a few of our packages now, so adding to the root of our project seems like the right approach going forward. I'll update the PR.

Reply:
sgtm